org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizard.java

```diff
@@ -101,10 +131,83 @@ public boolean performFinish() {
 				}
 			}
 		} catch (CoreException e) {
-			MessageDialog.openError(getShell(), "Unable to load project description", "");
+			MessageDialog.openError(getShell(), "Error", "Unable to load project description.");
 			return false;
 		}
 		return true;
 	}
 
+	private Set<IWorkingSet> getWorkingSets(Object o) {
```

Comment:
Here, the code is only supposed to run for an `IResource`, so instead of passing an object, it would be simpler to pass an IResource.